WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptView.swift

```diff
@@ -30,13 +36,27 @@ struct POSSendReceiptView: View {
                 .font(POSFontStyle.posTitleRegular)
                 .focused()
                 .padding()
+            if let errorMessage = errorMessage {
+                Text(errorMessage)
+                    .font(POSFontStyle.posBodyRegular)
+                    .foregroundColor(.red)
+            }
 
             Button(action: {
                 Task { @MainActor in
+                    guard isEmailValid else {
+                        errorMessage = Localization.emailValidationErrorText
+                        return
+                    }
                     isLoading = true
-                    await posModel.sendReceipt(to: textFieldInput)
+                    do {
+                        try await posModel.sendReceipt(to: textFieldInput)
```

Comment:
`.onSubmit` could also be set on `Text` so the keyboard "Return" key would send the email as well. A helpful UX.

Reply:
That's a great idea, I didn't know about `.onSubmit`. Updated on https://github.com/woocommerce/woocommerce-ios/pull/14651/commits/b98951030ed254060dab458b6b56772073df5d6e